include/deal.II/base/utilities.h

```diff
@@ -377,6 +390,32 @@ namespace Utilities
   std::vector<unsigned long long int>
   invert_permutation (const std::vector<unsigned long long int> &permutation);
 
+  /**
+   * Given an arbitrary object of type T, use boost::serialization utilities
+   * to pack the object into a vector of characters. The object can be unpacked
+   * using the Utilities::unpack function below.
+   *
+   * If the library has been compiled with ZLIB enabled, then the output buffer
+   * is compressed.
+   *
+   * @author Luca Heltai, 2017.
```

Comment:
since you moved the fucntion, maybe add @tjhei as an author as IIRC he wrote those originally.

Reply:
@tjhei wrote the initial function, I added the compression. Either way, though, great idea to externalize this into a separate set of functions!

Reply:
> great idea to externalize this into a separate set of functions!

👍